Providers/CoreServiceProvider.php

```diff
@@ -244,4 +247,50 @@ private function getCentralisedTranslationPath(Module $module)
     {
         return $this->app['modules']->find('Translation')->getPath() . "/Resources/lang/{$module->getLowerName()}";
     }
+
+    /**
+     * List of Custom Blade Directives
+     */
+    public function bladeDirectives()
+    {
+        /**
+         * Set variable.
+         * Usage: @set($variable, value)
+         */
+        Blade::directive('set', function ($expression) {
+            list($variable, $value) = $this->getArguments($expression);
+
+            return "<?php {$variable} = {$value}; ?>";
+        });
+
+        /**
+         * Php explode()
+         * Usage: @explode($delimiter, $string)
+         */
+        Blade::directive('explode', function ($expression) {
+            list($delimiter, $string) = $this->getArguments($expression);
+
+            return "<?php echo explode({$delimiter}, {$string}); ?>";
+        });
+
+        /**
+         * php implode()
+         * Usage: @implode($delimiter, $array)
+         */
+        Blade::directive('implode', function ($expression) {
+            list($delimiter, $array) = $this->getArguments($expression);
+
+            return "<?php echo implode({$delimiter}, {$array}); ?>";
+        });
+    }
+
+    /**
+     * Get argument array from argument string.
+     * @param $argumentString
+     * @return array
+     */
+    private function getArguments($argumentString)
+    {
+        return explode(', ', str_replace(['(', ')'], '', $argumentString));
```

Comment:
Wasn't aware of that usage of `str_getcsv` in this case, interesting!  Maybe a good idea to use that, so that it's easily refactored on 5.3